### PR TITLE
Add ability to pass in global_variables, such as the app_name, into the ...

### PR DESCRIPTION
**Pull request by chrisndodge**
...Backbone views

### CODE REVIEW

**Comment by chrisndodge** [comment]
@afzaledx @muhhshoaib I created a branch off of your branch to be able for the application host container to pass along global variables, such as 'app_name' all the way through the Backbone views. Can you look at this and I'll merge it into your working branch.

File: edx_notifications/server/web/utils.py [hunk 1]
```diff
@@ -40,10 +40,16 @@ def get_notifications_widget_context():
...
+            'app_name': 'Your App Name Here',
+        },
         'view_templates': {
             'notification_icon': get_template_path('notification_icon.html'),
             'notification_pane': get_template_path('notification_pane.html'),
         }
     }
 
+    if override_context:
+        context.update(override_context)
```
Comment (muhhshoaib) [inline edx_notifications/server/web/utils.py]: @chrisndodge what is override_context here. What i am getting is that some extra settings that may be needed in the future ?

**Comment by muhhshoaib** [comment]
looks good :+1:

File: edx_notifications/server/web/utils.py (hunk 1 above)
Comment (chrisndodge) [inline edx_notifications/server/web/utils.py]: This is where an application can pass in additional context, for example, the app_name which only the application tier knows about.

**Pull request merged**